internal/alias/config.go

```diff
@@ -0,0 +1,129 @@
+package alias
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"github.com/ghodss/yaml"
+	"github.com/scaleway/scaleway-sdk-go/scw"
+)
+
+const (
+	scwAliasConfigPath = "SCW_ALIAS_CONFIG_PATH"
+
+	defaultConfigFileName   = "alias.yaml"
+	defaultConfigPermission = 0600
+)
+
+type Config struct {
+	// Aliases are raw aliases that allow to expand a command
+	// "scw instance sl", sl may be an alias and would expand command
+	// "scw instance server list"
+	Aliases map[string][]string `json:"aliases"`
+	// ResourceAliases are aliases specific to a resource
+	// it allows to add an alternative name to a namespace, resource or verb
+	ResourceAliases map[string][]string `json:"resources"`
```

Comment:
can some arguments be passed to? It could be useful to pass a given profile or specific configuration for a given command

Reply:
profile cannot be passed as this is a flag and not an argument but arguments can be passed:
- `scw alias create devserver command.0=instance command.1=server command.2=create command.3="type=PLAY2-MICRO" command.4=tags.0=devserver`
After you can use this alias with other arguments
- `scw devserver image=debian_buster`
You also get completion
- `scw devserver <TAB>` => `additional-volumes.0= boot-type= image= ipv6=`